SPECS/xcp-ng-release.spec

```diff
@@ -286,64 +286,57 @@ EOF
  #### RULES ####
 EOF
 
-%triggerin config -- openssh-server
-# XCP-ng, 2023-01-31
-# We diverged here from xenserver-release-8.2.1-10: they chose to fully replace the sshd and ssh configuration each time either xenserver-release-config or openssh-server is updated.
-# We would like not to overwrite user configurations this way, so keeping the patches for XCP-ng 8.2.x
+%triggerin config -- openssh-server >= 7.4p1-23.2.1
+# XCP-ng, 2024-05-13
+# Starting from openssh 7.4p1-23.2.1, the default ciphers, MACs, KexAlgorithms,
+# and HostKeyAlgorithms are now compiled in openssh code. This triggerin is used
+# to remove them from sshd_config. This is easier to do it through this package
+# rather than from the openssh one because this would be reverted if openssh is
+# upgraded without unpgrading xcp-ng-release.
```

Comment:
A rewording proposal:

```suggestion
# to remove them from sshd_config. This is easier to do it through this package
# rather than from the openssh-server one to avoid conflicting triggers (ordering issue)
# between an old xcp-ng-release-config and a newer openssh-server package.
```

Reply:
Done